app/models/conference.rb

```diff
@@ -858,7 +858,7 @@ def cfp_set?
   # * +True+ -> If conference has a start and a end date.
   # * +False+ -> If conference has no start or end date.
   def registration_date_set?
-    !!registration_period && !!registration_period.start_date && !!registration_period.end_date
+    registration_period && registration_period.start_date && registration_period.end_date
```

Comment:
Sorry, it was needed because it not it return the last value, so `registration_period.end_date`

You can write it like that:
`
registration_period && registration_period.start_date.present? && registration_period.end_date.present?`

That's why the tests are failing.